tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<commonviews:ToolPage
+    x:Class="DevHome.RepositoryManagement.Views.RepositoryManagementMainPageView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:commonCustomControls="using:DevHome.Common.Environments.CustomControls"
+    xmlns:commonviews="using:DevHome.Common.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:DevHome.RepositoryManagement.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:viewmodels="using:DevHome.RepositoryManagement.ViewModels"
+    behaviors:NavigationViewHeaderBehavior.HeaderMode="Never"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+    <commonviews:ToolPage.Resources>
+        <ResourceDictionary>
+            <Style x:Key="ListViewItemStretchStyle" TargetType="ListViewItem">
+                <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+                <Setter Property="Margin" Value="0" />
+                <Setter Property="Padding" Value="0" />
+            </Style>
+            <Style x:Key="HorizontalThreeDotsStyle" TargetType="Button">
+                <Setter Property="Background" Value="Transparent" />
+                <Setter Property="Content" Value="&#xE712;" />
```

Comment:
Consider using a theme resource instead of a color value.

Reply:
Changed to `ControlFillColorTransparent`